azurerm/provider.go

```diff
@@ -100,6 +100,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_container_registry":          resourceArmContainerRegistry(),
 			"azurerm_container_service":           resourceArmContainerService(),
 			"azurerm_container_group":             resourceArmContainerGroup(),
+			"azurerm_managed_cluster":             resourceArmManagedCluster(),
```

Comment:
taking a look around, I'm seeing several names for this Product: `Azure Kubernetes Service (AKS)` and `Azure Container Services (Managed)`, or are they both the same thing?

Reply:
Yes, both same service. AKS isn't really an acronym though. It is usually mentioned as `Azure Container Service (AKS)`, and in the portal it shows as `Azure Container Service (managed)`